geode-core/src/distributedTest/java/org/apache/geode/internal/jta/dunit/LoginTimeOutDUnitTest.java

```diff
@@ -263,15 +263,15 @@ public static void runTest1() throws Exception {
         logger.debug("Naming Exception caught in lookup: " + e);
         fail("failed in naming lookup: " + e);
       } catch (Exception e) {
-        logger.debug("Exception caught during naming lookup: {}", e);
+        logger.debug("Exception caught during naming lookup:", e);
         fail("failed in naming lookup: " + e);
       }
       try {
         for (int count = 0; count < MAX_CONNECTIONS; count++) {
           ds.getConnection();
         }
       } catch (Exception e) {
-        logger.debug("Exception caught in runTest1: {}", e);
+        logger.debug("Exception caught in runTest1:", e);
```

Comment:
Is it possible that these log messages intended to log the message only and not the stacktrace?

Reply:
debug logs, I think the whole stack is better than the message

Reply:
Also, its just a test, should not matter much